package.json

```diff
@@ -18,7 +18,7 @@
     "node": ">= 8"
   },
   "dependencies": {
-    "@sentry/cli": "^2.0.2"
+    "@sentry/cli": "^1.74.4"
```

Comment:
This PR is downgrading `@sentry/cli` from v2 to v1. Is this intentional?
The [release notes say that the major version was upgraded](https://github.com/getsentry/sentry-webpack-plugin/releases/tag/v1.18.9), but the reality is that in `1.18.9` release we are still stuck with `@sentry/cli` v1 😕 
cc @sawilde

Reply:
Thanks for pointing that out, changelog is incorrect; updated it.
We stick to v1 for now, as v2 has minimal version of node set to 12, which is not the case for this plugin.
We'll update major once we decide to move the plugin to the next major as well.

Reply:
nw - thanks for the update 

would the alternative be - as horrid it may sound - to have two branches one supporting v1 and the other v2 of @sentry/cli? I've seen other packages do something similar but not sure on how maintainable it would be for the team.

Reply:
We are definitely going to have v1 and v2, however, it still requires doing some work and major release, which I don't have enough capacity for right now.
This is precisely how we handle this in sentry-cli, and why we can stick to v1 here:
- https://github.com/getsentry/sentry-cli/tree/1.x
- https://github.com/getsentry/sentry-cli/tree/master